### ISSUE TITLE
Alternative product family

### ISSUE THREAD

**Issue by SushmaDG** [manikantapujar/ROS-Survo#5]
![screenshot from 2016-11-03 17-41-44](https://cloud.githubusercontent.com/assets/22175791/19965574/a4b2020c-a1ed-11e6-8106-10f4b8ca2233.png)

**Comment by manikantapujar** [comment]
From position update and planning function block the signal are transfered  to motors 
can we add signal line from that block to motor block

**Comment by SushmaDG** [comment]
When the position is updated it is reflected in the map created, so signal line doesn't go directly to the motors.

**Comment by manikantapujar** [comment]
so the new position must be updated to motors also to move to new position

**Comment by SushmaDG** [comment]
yes but it doesn't happen directly, instead through the map. It forms a loop with mapping and navigation blocks, after that motor will rotate. It cannot be given directly. As u said the updated position of motor will be recorded by encoder so loop exists there.

**Comment by manikantapujar** [comment]
k
